packages/contracts/deploy/helpers.ts

```diff
@@ -158,170 +140,6 @@ export async function detemineDeployerNextAddress(
-    // Either a frontrun happened or something else. Thus we abort here
-    throw new Error(`${subdomain} is already present! Aborting...`);
-  }
-
-  const pluginRepoFactoryAddress = await getContractAddress(
-    'PluginRepoFactory',
-    hre
-  );
-
-  const pluginRepoFactoryFactory = new PluginRepoFactory__factory(signers[0]);
-  const pluginRepoFactoryContract = pluginRepoFactoryFactory.attach(
-    pluginRepoFactoryAddress
-  );
-
-  const {deployer} = await hre.getNamedAccounts();
-
-  const tx = await pluginRepoFactoryContract.createPluginRepo(
-    subdomain,
-    deployer
-  );
-  console.log(
-    `Creating & registering repo for ${pluginName} with tx ${tx.hash}`
-  );
-  await tx.wait();
-
-  const event = await findEventTopicLog<PluginRepoRegisteredEvent>(
-    tx,
-    PluginRepoRegistry__factory.createInterface(),
-    'PluginRepoRegistered'
-  );
-  const repoAddress = event.args.pluginRepo;
-
-  hre.aragonPluginRepos[pluginName] = repoAddress;
-
-  console.log(
-    `Created & registered repo for ${pluginName} at address: ${repoAddress}` //, with contentURI ${ethers.utils.toUtf8String(releaseMetadata)}`
-  );
-}
-
-export async function createVersion(
-  pluginRepoContract: string,
-  pluginSetupContract: string,
-  releaseNumber: number,
-  releaseMetadata: string,
-  buildMetadata: string
-): Promise<void> {
-  const signers = await ethers.getSigners();
-
-  const PluginRepo = new PluginRepo__factory(signers[0]);
-  const pluginRepo = PluginRepo.attach(pluginRepoContract);
-
-  const tx = await pluginRepo.createVersion(
-    releaseNumber,
-    pluginSetupContract,
-    buildMetadata,
-    releaseMetadata
-  );
-
-  console.log(`Creating build for release ${releaseNumber} with tx ${tx.hash}`);
-
-  await tx.wait();
-
-  const versionCreatedEvent = await findEvent<VersionCreatedEvent>(
-    tx,
-    'VersionCreated'
-  );
-
-  console.log(
-    `Created build ${versionCreatedEvent.args.build} for release ${
-      versionCreatedEvent.args.release
-    } with setup address: ${
-      versionCreatedEvent.args.pluginSetup
-    }, with build metadata ${ethers.utils.toUtf8String(
-      buildMetadata
-    )} and release metadata ${ethers.utils.toUtf8String(releaseMetadata)}`
-  );
-}
-
-export type LatestVersion = {
-  versionTag: VersionTag;
-  pluginSetupContract: string;
-  releaseMetadata: string;
-  buildMetadata: string;
-};
-
-function isSorted(latestVersions: LatestVersion[]): boolean {
-  // The list of latest versions has to start with the first release, otherwise something is wrong and we must stop.
-  if (latestVersions[0].versionTag[0] != 1) {
-    return false;
-  }
-
-  for (let i = 0; i < latestVersions.length - 1; i++) {
-    if (
-      !(
-        latestVersions[i + 1].versionTag[0] ==
-        latestVersions[i].versionTag[0] + 1
-      )
-    ) {
-      return false;
-    }
-  }
-  return true;
-}
-
-export async function populatePluginRepo(
```

Comment:
Just a note that we have a task to move the `PlaceholderSetup` functionality to the osx-hardhat-plugin-template https://aragonassociation.atlassian.net/browse/OS-1078